modules/samtools/bamtocram/meta.yml

```diff
@@ -0,0 +1,52 @@
+name: samtools_bamtocram
+description: filter/convert and then index CRAM file
+keywords:
+  - view
+  - index
+  - bam
+  - cram
+tools:
+  - samtools:
+      description: |
+        SAMtools is a set of utilities for interacting with and post-processing
+        short DNA sequence read alignments in the SAM, BAM and CRAM formats, written by Heng Li.
+        These files are generated as output by short read aligners like BWA.
+      homepage: http://www.htslib.org/
+      documentation: hhttp://www.htslib.org/doc/samtools.html
+      doi: 10.1093/bioinformatics/btp352
+      licence: ["MIT"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - input:
+      type: file
+      description: BAM/SAM file
+      pattern: "*.{bam,sam}"
+  - index:
+      type: file
+      description: BAM/SAM index file
+      pattern: "*.{bai,sai}"
+  - fasta:
+      type: file
+      description: Reference file to create the CRAM file
+      pattern: "*.{fasta,fa}"
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - cram_crai:
+      type: file
+      description: filtered/converted CRAM file + index
+      pattern: "*{.cram,.crai}"
```

Comment:
I thought these should be listed as separate elements (even though they are output as a single tuple?)

Reply:
do you want me to make a PR for it?